lib/core/src/utils.rs

```diff
@@ -59,6 +64,22 @@ pub(crate) fn deserialize_tx_hex(tx_hex: &str) -> Result<Transaction> {
     )?)?)
 }
 
+pub(crate) async fn derive_fee_rate(
+    wallet: Arc<dyn OnchainWallet>,
+    amount_sat: u64,
+    recipient_address: &str,
+    absolute_fees: u64,
+) -> Result<f32> {
+    let standard_fees = wallet
+        .build_tx(None, recipient_address, amount_sat)
+        .await?
+        .all_fees()
+        .values()
+        .sum::<u64>() as f32;
+
+    Ok(0.1 * absolute_fees as f32 / standard_fees)
```

Comment:
Is the `0.1` actually the base feerate that's being "adjusted" with the division? If so, then why not use `STANDARD_FEE_RATE_SAT_PER_VBYTE` constant instead, so it's clear what it represents and it can be replaced easily in the future?

Reply:
Agreed, will do